app/serializers/forest_liana/serializer_factory.rb

```diff
@@ -108,7 +108,9 @@ def relationship_related_link(attribute_name)
 
           if ret[:href].blank?
             begin
-              relationship_records = object.send(attribute_name)
+              if @options[:include].try(:include?, attribute_name.to_s)
+                relationship_records = object.send(attribute_name)
+              end
```

Comment:
This change breaks the hasMany retrieval in a record detail because `href` are not computed anymore:
<img width="797" alt="screen shot 2016-11-03 at 10 44 55" src="https://cloud.githubusercontent.com/assets/664261/19978046/d4b8ab1c-a1b2-11e6-98c1-1fd7b9e8e6e1.png">